assets/templates/99_registry.yaml

```diff
@@ -15,6 +15,4 @@ spec:
     write:
       maxWaitInQueue: 0s
   storage:
-    filesystem:
-      volumeSource:
-        emptyDir: {}
+    emptyDir: {}
```

Comment:
This changed: https://github.com/openshift/cluster-image-registry-operator/commit/cc4403d6e

Reply:
Let's get this useful context in the commit message - it'll get lost here after the PR is merged

Reply:
Done